src/ast.js

```diff
@@ -212,9 +215,9 @@ function setContext (c, e, ctx, n) {
         case Sk.astnodes.Compare:
             exprName = "comparison";
             break;
-        case Sk.astnodes.Repr:
-            exprName = "repr";
-            break;
+        // case Sk.astnodes.Repr:
```

Comment:
Do we not need this for backwards compat with py2

Reply:
I don't know.  I do know that it is not defined in Python 2 or Python 3, so if we do "need" it, it wasn't being used before this pull request.  Removing it changed nothing in the code base, as it was dead code that could never execute.

Reply:
Yeah you're right, I think it was left inplace because the python 2 ast/compiler does something special for repr. I'll try and find out. but it seems indeed best to comment this out for now.